apps/estuary/config/dev.exs

```diff
@@ -0,0 +1,5 @@
+use Mix.Config
+
+config :estuary,
+  elsa_endpoint: [localhost: 9092],
+  event_stream_topic: "event-stream"
```

Comment:
this probably belongs in either config.exs or integration.exs

Reply:
Keywords can be tricky since they get merged when you layer them from different config files (config.exs, prod.exs, runtime.exs for instance). It may be safer to keep these values out of the root config file unless you know they're not likely to change.